samples/notebooks/Lightweight Python components - sharing 1 - produce component files.ipynb

```diff
@@ -0,0 +1,155 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Lightweight python components - share component files\n",
+    "\n",
+    "Lightweight python components do not require you to build a new container image for every code change.\n",
+    "They're intended to use for fast iteration in notebook environment.\n",
+    "\n",
+    "#### Building a lightweight python component\n",
+    "To build a component just define a stand-alone python function and then call kfp.components.func_to_container_op(func) to convert it to a component that can be used in a pipeline.\n",
+    "\n",
+    "There are several requirements for the function:\n",
+    "* The function should be stand-alone. It should not use any code declared outside of the function definition. Any imports should be added inside the main function. Any helper functions should also be defined inside the main function.\n",
+    "* The function can only import packages that are available in the base image. If you need to import a package that's not available you can try to find a container image that already includes the required packages. (As a workaround you can use the module subprocess to run pip install for the required package.)\n",
```

Comment:
Maybe add a link to the sample notebook which demonstrates how to build a container inside cluster? Or even include some code snippets?